.changeset/brave-rats-fetch.md

```diff
@@ -0,0 +1,5 @@
+---
+'@farfetched/react': minor
```

Comment:
It is not minor, it is a fix of a bug. So, it is patch 🤗

```suggestion
'@farfetched/react': patch
```

Reply:
So, now it is two changes 🤗 old one and new one. I assume, we can delete one of them.